sdks/nodejs/src/agonesSDK.js

```diff
@@ -19,7 +19,26 @@ const services = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
-		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
+		function port() {
+			const defaultPort = 59357;
+			const port = process.env.AGONES_SDK_GRPC_PORT;
+			if (port === undefined) {
+				console.log("Environment variable AGONES_SDK_GRPC_PORT not defined, using default port ${defaultPort}");
```

Comment:
I think the linter should fix the quotes here to avoid having mixed quotes. If you run `npm run lint` that should do the trick.
It would be worth thinking about running lint and tests/coverage automatically

Reply:
Actually I think you may have intended to use a backquote here in order to use variable substitution

Reply:
I've switched to backticks and also added single quotes around the string to make it stand out more in the console output.